registry_module_test.go

```diff
@@ -226,6 +227,8 @@ func TestRegistryModulesCreateWithVCSConnection(t *testing.T) {
 			OAuthTokenID:      oauthTokenTest.ID,
 			DisplayIdentifier: githubIdentifier,
 			IngressSubmodules: true,
+			RepositoryHTTPURL: fmt.Sprintf("https://github.com/%s", githubIdentifier),
+			ServiceProvider:   "github",
```

Comment:
See [comment](https://github.com/hashicorp/go-tfe/pull/191/files#r592339380). Could we make this an explicit const?

Reply:
Ah thanks to @lafentres for pointing me to here https://github.com/hashicorp/go-tfe/blob/master/oauth_client.go#L42 where we already have them

Reply:
Also could there be a version of this that supports github enterprise?